src/components/SignalCard/controls/MoreSignalActions.js

```diff
@@ -44,21 +46,35 @@ const MoreSignalActions = ({
   isPublic,
   isUserTheAuthor,
   deleteEnabled = true,
-  editable = true
+  editable = true,
+  signal
 }) => {
   const canShare = true
 
   if (!isUserTheAuthor) {
     return (
-      canShare && (
-        <ShareSignal
-          className={styles.shareBtn}
-          signalId={signalId}
-          signalTitle={signalTitle}
-          shareBtnClassName={styles.shareSingle}
-          trigger={ShareIcon}
-        />
-      )
+      <div className={styles.buttonWrapper}>
+        {canShare && (
+          <ShareSignal
+            className={styles.shareBtn}
+            signalId={signalId}
+            signalTitle={signalTitle}
+            shareBtnClassName={styles.shareSingle}
+            trigger={ShareIcon}
+          />
+        )}
+        <DesktopOnly>
+          <CopySignal
+            signal={signal}
+            label='Copy to my alerts'
+            as='div'
+            classes={{
+              copyBtn: styles.buttonWrapper__button
+            }}
+            btnParams={{ variant: 'flat' }}
```

Comment:
Since `classes` and `btnParams` are constant, it's possible to move them out of render function